.github/workflows/cypress.yml

```diff
@@ -0,0 +1,16 @@
+name: cypress-test-run
+on: [push]
```

Comment:
To me this is good, but if we reach some limits or anything, we can consider alternatives.

Reply:
Yeah, we can check the https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows 
and choose what suits us better